external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MariaDBKrbIntegrationSuite.scala

```diff
@@ -30,7 +30,7 @@ class MariaDBKrbIntegrationSuite extends DockerKrbJDBCIntegrationSuite {
   override protected val keytabFileName = "mariadb.keytab"
 
   override val db = new DatabaseOnDocker {
-    override val imageName = "mariadb:10.4"
+    override val imageName = sys.env.getOrElse("MARIADB_DOCKER_IMAGE_NAME", "mariadb:10.4")
```

Comment:
Is this the latest one?

Reply:
This is not changed, but I failed. So, cc @gaborgsomogyi 
```
MariaDBKrbIntegrationSuite:
org.apache.spark.sql.jdbc.MariaDBKrbIntegrationSuite *** ABORTED ***
  Exception encountered when invoking run on a nested suite - The code passed to eventually never returned normally. Attempted 26 times over 2.269450662233333 minutes. Last failure message: Could not connect to address=(host=192.168.1.114)(port=63328)(type=master) : Socket fail to connect to host:192.168.1.114, port:63328. connect timed out. (DockerJDBCIntegrationSuite.scala:159)
```

Reply:
This test with the latest one, 10.5, failed consistently, but 10.4 looks fine on my local env.

Reply:
Could you check this, @gaborgsomogyi ?

Reply:
Started to have a look...

Reply:
I've executed the mentioned test with this PR but it works fine on my machine.
<img width="397" alt="Screenshot 2020-10-02 at 10 23 09" src="https://user-images.githubusercontent.com/18561820/94904123-1d1fd700-049b-11eb-8275-74a0f2cc3fd1.png">
What I can imagine is that the docker version is different. When I've started the development in this area I remember there was quite a stuggle to install a working version. I'm still not on the latest.
<img width="373" alt="Screenshot 2020-10-02 at 10 17 49" src="https://user-images.githubusercontent.com/18561820/94904337-7425ac00-049b-11eb-8e3e-f1f8b0b37062.png">

Reply:
Ah, ok. Thanks for the check. I will re-run it on the latest one.

Reply:
I've had a look at why `10.5` is failing. Please see that the gssapi plugin version is pinned to a specific version: https://github.com/apache/spark/blob/9b88aca2954cd931c94a7cc788c3c3f7a33e99b7/external/docker-integration-tests/src/test/resources/mariadb_docker_entrypoint.sh#L21

When the version changed to `1:10.5.5+maria~focal` then it works like charm :)

Reply:
Oh, I see! Thanks a lot! I'll check and update it.